cpp/thirdparty/README.md

```diff
@@ -23,3 +23,19 @@ See the "Build Dependency Management" section in the [C++ Developer
 Documentation][1].
 
 [1]: https://github.com/apache/arrow/blob/main/docs/source/developers/cpp/building.rst
+
+## Update versions automatically
+
+There is a convenient script that update versions in `versions.txt` to
+the latest version automatically. You can use it like the following:
+
+```console
+dev/release/update.rb PRODUCT_PATTERN1 PRODUCT_PATTERN2 ...
```

Comment:
This is neat, but the path doesn't look right, should be `cpp/thirdparty/update.rb`?

Reply:
Oh... Good catch! You're right.